src/static/css/pages/careers.less

```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```

Comment:
Actually this PR should include the size change.
Can you please change:

```less
.students-and-graduates .media_image {
    width: 150px
}
```

…to…

```less
// TODO: Consolidate site-wide media_image responsive sizes into one class.
.students-and-graduates .media_image {
         width: 130px;
        .respond-to-min(800px, {
            width: 150px;
        });
}
```